moveit_planners/chomp/chomp_interface/include/chomp_interface/chomp_planning_context.h

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
Because CHOMP used the very old and now deprecated Planner interface from moveit.